sf-docker/all-in-one/docker-compose.yml

```diff
@@ -781,6 +821,8 @@ services:
     ports:
       - 8000:8000
     env_file: .env.sfstation
+    volumes:
+      - "lic-volume:/etc/innovatrics/"
```

Comment:
this seems to be incorrect. likely to fix it needs to be like this:
volumes:
      - ./iengine.lic:/etc/innovatrics/iengine.lic

Reply:
fixed